common/transport/amqp/src/amqp.ts

```diff
@@ -213,11 +214,16 @@ export class Amqp {
             this._fsm.transition('connecting', connectionParameters, connectCallback);
           },
           disconnect: (callback) => callback(null, new results.Disconnected()),
+          // @ts-ignore TS6133 endpoint and linkOptions are not used
```

Comment:
would be in favor of not having these and instead using _

Reply:
Great idea.  Thank you.